desktop/libs/indexer/src/indexer/indexers/sql.py

```diff
@@ -332,6 +348,17 @@ def create_table_from_local_file(self, source, destination, start_time=-1):
                   'table_name': table_name,
                   'csv_rows': csv_rows
                 }
+        if dialect == 'phoenix':
+          for csv_row in list_of_tuples:
```

Comment:
PS: BTW `list_of_tuples` not a good name

Reply:
done